python/tvm/meta_schedule/search_strategy/evolutionary_search.py

```diff
@@ -58,7 +58,7 @@ class EvolutionarySearch(SearchStrategy):
     def __init__(
         self,
         *,
-        population_size: int = 2048,
+        population_size: int = 512,
```

Comment:
cc @zxybazh This change so small and looks unrelated to this PR, but it has huge implications. Was there enough discussion to make this change? Tuning time will become much shorter, which I like, but could there be a concern for perf regression due to this change?

Reply:
@masahi Ask for asking, and it's definitely a valid concern. The reason is that we found it's somehow set to an unreasonably large number (we were consistently using 512 on downstream), and now wanted to scale it back. Also note that the population calculation is slightly different than AutoScheduler, which includes invalid candidates in population, while ours always prune ahead of time.

We did make sure we have proper numbers in hand before merging, and please refer to the table below for details:

|                     | Mainline (ms) | This PR (ms) | Difference |
|---------------------|---------------|--------------|------------|
| resnet_50      |   1.829128008 |   1.73111331 |      5.66% |
| mobilenet_v2   |  0.4773168361 | 0.4806015715 |     -0.68% |
| resnet_18      |   0.678600832 | 0.6301909627 |      7.68% |
| mobilenet_v3   |  0.6649458484 |  0.668666894 |     -0.56% |
| wide_resnet_50 |   3.753781691 |  3.077549256 |     21.97% |
| densenet_121   |   2.361973117 |  2.288411393 |      3.21% |
| inception_v3   |    3.53431975 |  3.478368823 |      1.61% |
| resnet3d_18    |   7.791658449 |  7.465777971 |      4.36% |